app/src/main/java/org/wikipedia/page/references/ReferenceListDialog.kt

```diff
@@ -0,0 +1,116 @@
+package org.wikipedia.page.references
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import kotlinx.android.synthetic.main.dialog_reference_list.*
+import org.wikipedia.R
+import org.wikipedia.activity.FragmentUtil
+import org.wikipedia.page.ExtendedBottomSheetDialogFragment
+import org.wikipedia.page.LinkHandler
+import org.wikipedia.page.LinkMovementMethodExt
+import org.wikipedia.util.DimenUtil
+import org.wikipedia.util.L10nUtil
+import org.wikipedia.util.StringUtil
+import org.wikipedia.util.log.L
+import org.wikipedia.views.DrawableItemDecoration
+
+class ReferenceListDialog : ExtendedBottomSheetDialogFragment() {
+    interface Callback {
+        val references: Observable<References>
+        val linkHandler: LinkHandler
+    }
+
+    private val disposables = CompositeDisposable()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.dialog_reference_list, container)
+    }
+
+    override fun onStart() {
+        super.onStart()
+        BottomSheetBehavior.from(view!!.parent as View).peekHeight = DimenUtil
+                .roundedDpToPx(DimenUtil.getDimension(R.dimen.readingListSheetPeekHeight))
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        L10nUtil.setConditionalLayoutDirection(view, callback()!!.linkHandler.getWikiSite().languageCode())
```

Comment:
`getWikiSite()` can be changed to `wikiSite`

Reply:
ok!